lib/Doctrine/DBAL/Types/DecimalType.php

```diff
@@ -49,6 +49,6 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
      */
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
-        return (null === $value) ? null : $value;
+        return (null === $value) ? null : (string) $value;
```

Comment:
I remove this change and the test passes. Not sure why. I'm going to branch and rebase this PR with this change reverted for now and see if the tests pass on travis.